mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -112,6 +112,7 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	FIELD field_28264 GAME_EVENT Lnet/minecraft/class_2348;
 	FIELD field_28265 POSITION_SOURCE_TYPE Lnet/minecraft/class_2378;
 	FIELD field_28266 GAME_EVENT_KEY Lnet/minecraft/class_5321;
+	FIELD field_29076 FLOAT_PROVIDER_TYPE Lnet/minecraft/class_2378;
```

Comment:
Imo if we have these float providers in registries, I suggest raising the floatprovider pacakge out of math to just util

Reply:
I put it here because it is used directly in math, but I can do that too. Would like a second opinion